utils/index.js

```diff
@@ -82,54 +82,64 @@ const bufferToJSON = (content, filepath) => {
  *
  * @param {String} repoName - The full repo name (username/repository)
  * @param {Array<String>} files - A list of files
- * @return {Array<String>} - The files downloaded
+ * @param {Array<String>} dependencies - A list of zel dependencies
+ * @return {Object} - An object with a list of files and dependencies that were downloaded
  */
-const fetchFiles = (repoName, files) => {
-    if (!files) {
-        throw new Error('No files found.');
+const fetchAndWrite = (repoName, { files, dependencies }) => {
+    if (!files && !dependencies) {
+        throw new Error('No files or dependencies found.');
     }
 
-    files.forEach((file) => {
-        const downloadUrl = `https://raw.githubusercontent.com/${repoName}/master/${file}`;
+    if (files && files.length) {
```

Comment:
Could just be a personal preference, but what if we set `files` and `dependencies` to `[]` here by default, bypassing the need for if-blocks?

```js
if (!...) throw new Error();

files = files || [];
dependencies = dependencies || [];

files.forEach(...);
dependencies.forEach(...);

// or, faster:

(files || []).forEach(...);
(dependencies || []).forEach(...);
```